addons/sale_timesheet/models/account.py

```diff
@@ -90,13 +90,8 @@ def _check_can_write(self, values):
 
     @api.model
     def _timesheet_preprocess(self, values):
-        if values.get('task_id') and not values.get('account_id'):
-            task = self.env['project.task'].browse(values.get('task_id'))
-            if task.analytic_account_id:
-                values['account_id'] = task.analytic_account_id.id
-                values['company_id'] = task.analytic_account_id.company_id.id
-        values = super(AccountAnalyticLine, self)._timesheet_preprocess(values)
-        return values
+        # TODO: remove me in master
```

Comment:
Can you please remove the http links to external resources from your commit? This is a bad practice as those links are not eternal...